src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -239,7 +245,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     new JesBackendCall(this, workflowDescriptor, key, locallyQualifiedInputs, abortRegistrationFunction)
   }
 
-  override def engineFunctions: WdlStandardLibraryFunctions = new JesEngineFunctionsWithoutCallContext()
+  override def engineFunctions(interface: IOInterface): WdlStandardLibraryFunctions = new JesEngineFunctionsWithoutCallContext(interface)
```

Comment:
if I'm reading this right it means that technically one could have an interface being passed in for the incorrect backend?